lib/matplotlib/axes/_axes.py

```diff
@@ -1631,7 +1629,8 @@ def semilogx(self, *args, **kwargs):
     # @_preprocess_data() # let 'plot' do the unpacking..
     @docstring.dedent_interpd
     def semilogy(self, *args, **kwargs):
-        r"""Make a plot with log scaling on the `y` axis.
+        """
```

Comment:
Although not crucial, it would be nice to take advantage of this opportunity to fix the docstring of semilogy to exactly match the one of semilogx, save for x/y.  Right now there are a number of discrepancies.

Reply:
I have already fixed the docstring of semilogy to match the one of semilogx.